config/config_helpers.go

```diff
@@ -204,20 +206,14 @@ func getParentTfVarsDir(include *IncludeConfig, terragruntOptions *options.Terra
 
 func parseGetEnvParameters(parameters string) (EnvVar, error) {
 	envVariable := EnvVar{}
-	matches := HELPER_FUNCTION_GET_ENV_PARAMETERS_SYNTAX_REGEX.FindStringSubmatch(parameters)
-	if len(matches) < 2 {
-		return envVariable, errors.WithStackTrace(InvalidGetEnvParams(parameters))
-	}
+	parsed := parseParamList(parameters)
```

Comment:
Again, since we're able to describe the entire parameter list, with zero or more parameters, we should just use that regex to extract all the parameters, then throw errors if it doesn't have the right number of args.

That's exactly what `parseParamList` does, so we just call that.